src/style.css

```diff
@@ -1 +1,81 @@
-/* add styles here */
+@font-face {
+  font-family: 'Avenir Book';
+  src: url('./fonts/Avenir-Book.ttf') format('truetype');
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: 'Avenir Heavy';
+  src: url('./fonts/Avenir-Heavy.ttf') format('truetype');
+  font-weight: bold;
+}
+
+body {
+  font-family: 'Avenir Book', sans-serif;
```

Comment:
You have used a fallback font for the body tag, which is good practice. However, you should consider adding more fallback fonts. For instance, you can add 'Arial', 'Helvetica' as fallbacks. This will ensure that if 'Avenir Book' doesn't load, the browser will try to load 'Arial', then 'Helvetica', and finally 'sans-serif'.